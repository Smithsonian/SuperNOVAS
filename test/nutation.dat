
2450545.000000 TEST       S2 O0 A1: 33571493.852328 -19382133.499726 14110085.519513 

2450545.000000 TEST       S2 O1 A1: 33571493.852328 -19382133.499727 14110085.519513 

2450545.000000 TEST       S2 O2 A1: 33571493.852328 -19382133.499727 14110085.519513 

2450545.000000 SUN        S0 O0 A1:    -0.006750     0.004995     0.002339 

2450545.000000 SUN        S0 O1 A1:    -0.006750     0.004995     0.002339 

2450545.000000 SUN        S0 O2 A1:    -0.006750     0.004995     0.002339 

2451545.000000 TEST       S2 O0 A1: 33570742.072740 -19384237.424814 14108983.946058 

2451545.000000 TEST       S2 O1 A1: 33570742.072740 -19384237.424814 14108983.946058 

2451545.000000 TEST       S2 O2 A1: 33570742.072740 -19384237.424814 14108983.946058 

2451545.000000 SUN        S0 O0 A1:    -0.007138    -0.002643    -0.000921 

2451545.000000 SUN        S0 O1 A1:    -0.007138    -0.002643    -0.000921 

2451545.000000 SUN        S0 O2 A1:    -0.007138    -0.002643    -0.000921 

2452545.000000 TEST       S2 O0 A1: 33570568.547723 -19385303.373251 14107932.249474 

2452545.000000 TEST       S2 O1 A1: 33570568.547723 -19385303.373251 14107932.249474 

2452545.000000 TEST       S2 O2 A1: 33570568.547723 -19385303.373251 14107932.249474 

2452545.000000 SUN        S0 O0 A1:     0.000656    -0.004704    -0.002013 

2452545.000000 SUN        S0 O1 A1:     0.000656    -0.004704    -0.002013 

2452545.000000 SUN        S0 O2 A1:     0.000656    -0.004704    -0.002013 
