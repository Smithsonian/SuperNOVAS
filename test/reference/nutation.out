  -10000.000 A0:            nan            nan            nan 
       0.000 A0:  0.80184132266 -0.54822872492 -0.23768836414 
   10000.000 A0: -0.18771974095  0.90107619827  0.39092580341 
   10000.000 A0: -0.52908777639 -0.77858316453 -0.33745278305 
   10000.010 A0: -0.52908777465 -0.77858316669 -0.33745278080 
  -10000.000 A1:  -0.5289218  -0.7786818  -0.3374855 
       0.000 A1:   0.8018413  -0.5482287  -0.2376884 
   10000.000 A1:  -0.1877197   0.9010762   0.3909258 
   10000.000 A1:  -0.5290878  -0.7785832  -0.3374528 
   10000.010 A1:  -0.5290878  -0.7785832  -0.3374528 
